genesis-tools/genesis-populate/src/lib.rs

```diff
@@ -238,17 +238,17 @@ impl GenesisBuilder {
     }
 
     fn add_additional_account(&mut self, account_id: AccountId) -> Result<()> {
-        const TESTING_INIT_BALANCE: Balance = 1_000_000_000_000_000;
-        const TESTING_INIT_STAKE: Balance = 0;
+        let testing_init_balance: Balance = 10u128.pow(30);
```

Comment:
Looks like need to rerun the 80M account generation. it's 75M/80M accounts now :joy:

Reply:
Unless you will be re-using the same accounts for lots of very heavy computation you don't need to.